src/Tizen.Multimedia.Remoting/WebRTC/MediaSource.cs

```diff
@@ -213,6 +213,70 @@ public Size VideoResolution
             }
         }
 
+        /// <summary>
+        /// Gets or sets the video frame rate of the current media source.
+        /// </summary>
+        /// <remarks>
+        /// This API is only supported in video media source, especially <a cref="MediaCameraSource"/>,
+        /// <a cref="MediaScreenSource"/>, <a cref="MediaTestSource(MediaType)"/>.<br/>
```

Comment:
suggestion : MediaTestSource(MediaType) -> (MediaType) is it intended?
I think we could remove this.

Reply:
Done